### PR TITLE
add numbers to add module documentation

### LINKED ISSUE #1389: Improve the nf-core installation and add prettier
**Is your feature request related to a problem? Please describe.**
There is missing instruction regarding the installation of prettier even if it's something needed

**Describe the solution you'd like**
Add instructions in the nf-core tutorial

### PR DESCRIPTION
**Pull request by asthara10**
Close https://github.com/nf-core/modules/issues/1389

<a href="https://gitpod.io/#https://github.com/nf-core/nf-co.re/pull/1083"></a>

### CODE REVIEW

**Comment by Emiller88** [comment]
@asthara10 I think this got messed up by #1090. I'm going to rebase it.

**Review by Emiller88 — APPROVED** [review]
Looks great! Maybe we could use `a,b,c` in the future, so we can reference them such as `guideline 2b`

**Comment by Emiller88** [comment]
Markdown errors should be fixed by #1098

**Review by sateeshperi — APPROVED** [review]
lgtm

File: markdown/developers/modules.md [hunk 1]
```diff
@@ -306,9 +306,9 @@ Once you<span class="x x-first x-last"> are </span>familiar with the module subm
 
 The key words "MUST", "MUST NOT", "SHOULD", etc. are to be interpreted as described in [RFC 2119](https://tools.ietf.org/html/rfc2119).
 
-### General
+### 1. General
```
Comment (ewels) [inline markdown/developers/modules.md:309]: Personally I don't love having numbers in headings, for one it breaks all existing softlink URLs if we need to add a new heading. Also I think it's just a bit ugly 😅 

But if I'm the only one thinking this then I'm happy to go with the crowd 👍🏻 Only a very minor thing.

**Review by ewels — APPROVED** [review]

**Review by ewels — APPROVED** [review]
Edited to remove numbers in headings 👍🏻

**Pull request merged**